internal/runnerinstall/ecs.go

```diff
@@ -330,15 +347,91 @@ func (i *ECSRunnerInstaller) Uninstall(ctx context.Context, opts *InstallOpts) e
 			Services: []*string{service.ServiceArn},
 		})
 		if err != nil {
-			log.Debug("error waint for runner to deactivate: %s", err)
+			log.Debug("error waiting for runner to deactivate: %s", err)
 			ui.Output("Error waiting for Runner service to deactivate: %s", clierrors.Humanize(err), terminal.WithErrorStyle())
 			return err
 		}
 
 		s.Update("Runner uninstalled")
 	}
 	s.Done()
-	return nil
+
+	// TODO: Still attempt to delete the EFS volume if the ECS service
+	// uninstall fails
+	efsSvc := efs.New(sess)
+	marker := ""
+	done := false
+	var fileSystems []*efs.FileSystemDescription
+	for !done {
+		fileSystemsResp, err := efsSvc.DescribeFileSystems(&efs.DescribeFileSystemsInput{
+			Marker:   aws.String(marker),
+			MaxItems: aws.Int64(100),
+		})
+		if err != nil {
+			return err
+		}
+		if marker == *fileSystemsResp.Marker {
+			done = true
+		}
+		fileSystems = append(fileSystems, fileSystemsResp.FileSystems...)
+	}
+
+	var fileSystemId *string
+	for _, fileSystem := range fileSystems {
+		// Check if tags match ID, if so then delete things
+		for _, tag := range fileSystem.Tags {
+			if *tag.Key == "runner-id" && *tag.Value == opts.Id {
+				fileSystemId = fileSystem.FileSystemId
+				goto DeleteFileSystem
+			}
+		}
+	}
+
+DeleteFileSystem:
```

Comment:
Delete this label? It doesn't look like it's used

Reply:
It is used [here](https://github.com/hashicorp/waypoint/pull/3944/files#diff-af12170bdcf18b294209fa40b9af88dc690c261e30f331fc7f520b1798b5a385R385)!